guides/doc-Content_Management_Guide/topics/Managing_Custom_File_Type_Content.adoc

```diff
@@ -99,14 +99,16 @@ Use this procedure to configure a repository in a directory on the base system w
 
 To create a file type repository in a local directory, complete the following procedure:
 
-. Ensure the Server and Satellite Tools repositories are enabled:
+ifeval::["build" == "satellite"]
+. Ensure the Server and `{RepoRHEL7ServerSatelliteToolsProductVersion}` repositories are enabled:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
 --enable={RepoRHEL7ServerSatelliteToolsProductVersion}
 ----
-+
+endif::[]
+
 . Install the Pulp Manifest package:
 +
 [options="nowrap" subs="+quotes,attributes"]
```

Comment:
Hi @ehelms, in Satellite, the python-pulp-manifest package is available in satellite-tools repository. Do you know where does this package come from in Foreman? I cannot find the exact repo that provides it.

The python-pulp-manifest package was added to satellite-tools as part of this bug:
 https://bugzilla.redhat.com/show_bug.cgi?id=1488515#c16

Reply:
@ehelms, I have installed my own Foreman and found that this is provided by the pulp repo